tests/PSRule.Rules.Azure.Tests/Azure.Deployment.Tests.ps1

```diff
@@ -54,6 +54,27 @@ Describe 'Azure.Deployment' -Tag 'Deployment' {
             $targetNames = $ruleResult | ForEach-Object { $_.TargetObject.name };
             $targetNames | Should -BeIn 'good';
         }
+
+        It 'Azure.Deployment.SecureValue' {
+            $sourcePath = Join-Path -Path $here -ChildPath 'Tests.Bicep.9.json';
+            $data = Export-AzRuleTemplateData -TemplateFile $sourcePath -PassThru;
+            $result = $data | Invoke-PSRule @invokeParams -Name 'Azure.Deployment.SecureValue';
+            $filteredResult = $result | Where-Object { $_.RuleName -eq 'Azure.Deployment.SecureValue' };
+
+            # Fail
+            $ruleResult = @($filteredResult | Where-Object { $_.Outcome -eq 'Fail' });
+            $ruleResult | Should -Not -BeNullOrEmpty;
+            # $ruleResult.Length | Should -Be 1;
```

Comment:
Should this comment be removed?

Reply:
@verabe I think this one should be uncommented and updated.